ocs_ci/ocs/must_gather/must_gather.py

```diff
@@ -0,0 +1,164 @@
+
+        """
+        files = TYPE_LOG[self.type_log]
+        for file in files:
+            self.files_not_exist.append(file)
+            for dir_name, subdir_list, files_list in os.walk(self.root):
+                if file in files_list:
+                    self.files_path[file] = os.path.join(dir_name, file)
+                    self.files_not_exist.remove(file)
+                    break
+
+    def validate_file_size(self):
+        """
+        Validate the file is not empty
+
+        """
+        for path, subdirs, files in os.walk(self.root):
+            for file in files:
+                file_path = os.path.join(path, file)
+                if Path(file_path).stat().st_size == 0:
+                    logger.error(f"log file {file} empty!")
+                    self.empty_files.append(file)
+
+    def validate_expected_files(self):
+        """
+        Make sure all the relevant files exist
+
+        """
+        self.search_file_path()
+        for file, file_path in self.files_path.items():
+            if not Path(file_path).is_file():
+                self.files_not_exist.append(file)
+            elif Path(file_path).stat().st_size == 0:
+                self.empty_files.append(file)
+            elif re.search(r'\.yaml$', file):
+                with open(file_path, 'r') as f:
+                    if 'kind' not in f.read().lower():
+                        self.files_content_issue.append(file)
+
+    def compare_running_pods(self):
+        """
+        Compare running pods list to "/pods" subdirectories
+
+        """
+        must_gather_helper = re.compile(r'must-gather-.*.-helper')
+        pod_objs = get_all_pods(namespace='openshift-storage')
+        pod_names = []
+        for pod in pod_objs:
+            if not must_gather_helper.match(pod.name):
+                pod_names.append(pod.name)
+
+        for dir_name, subdir_list, files_list in os.walk(self.root):
+            if re.search('openshift-storage/pods$', dir_name):
+                pod_path = dir_name
+
+        pod_files = []
+        for pod_file in os.listdir(pod_path):
+            if not must_gather_helper.match(pod_file):
+                pod_files.append(pod_file)
+
+        assert set(sorted(pod_files)) == set(sorted(pod_names)), (
+            f"List of openshift-storage pods are not equal to list of logs "
+            f"directories list of pods: {pod_names} list of log directories: {pod_files}"
+        )
+
+    def cleanup(self):
+        """
+        Delete temporary folder.
+
+        """
+        if re.search('_ocs_logs$', self.temp_folder):
+            shutil.rmtree(
+                path=self.temp_folder,
+                ignore_errors=False,
+                onerror=None
+            )
+
+    def print_invalid_files(self):
+        """
+        Print Invalid Files
+
+        """
+        error = ''
+        if len(self.empty_files) + len(self.files_not_exist) + len(
+            self.files_content_issue
+        ) != 0:
```

Comment:
No need to get the length or compare with 0. Any empty lists will evaluate to `False` and we can check them all with `any`.

```suggestion
        if any([self.empty_files, self.files_not_exist, self.files_content_issue]):
```